src/utils/tensor.js

```diff
@@ -28,27 +30,40 @@ const DataTypeMap = new Map([
     ['int32', Int32Array],
     ['uint32', Uint32Array],
     ['int64', BigInt64Array],
-])
+]))
 
 /**
  * @typedef {import('./maths.js').AnyTypedArray | any[]} DataArray
  */
 
 const ONNXTensor = ONNX.Tensor;
```

Comment:
Only used as type for the constructor, but stripped from type definitions, so it will be unresolved/any, we can just remove the line